examples/covariance/plot_outlier_detection.py

```diff
@@ -18,6 +18,9 @@
   hence more adapted to large-dimensional settings, even if it performs
   quite well in the examples below.
 
+- using the Local Outlier Factor to measure the local deviation of a given
```

Comment:
Yes very weird! it is the folder of the first outlier detection algorithm in scikit-learn.